# Bug 011

## Bug Report

This bug would happend when typing letters into the shell. It would echo the first letter without error, but its not supposed to do that. With the addition of string support, letters are now defined so this just causes a name error, because literals apply to functions and variables, and currently none exists.

old

```py
>>> testing
t
```

new

```py
>>> testing
NameError: name 'testing' is not defined
```

## Update

Bug is reopen, and currently being fixed.

___
